### DISCUSSION TITLE
Custom messagges death

### DISCUSSION THREAD

**Discussion by Guest1131** [EssentialsX/Essentials#4159] [Ideas and Feature Suggestions] [answered]
I saw that essentials, doesn't have any entry in the config, about deaths

I would like you to add in essentialsX a custom message for death.

For example in pvp: "Player has killed the player with the diamond sword" where in the message is indicated the object with which he has been killed.

**Comment by JRoy** [comment, marked as answer]
Closed in favor of #4158
